### ISSUE TITLE
Migration rollback with foreign keys

### ISSUE THREAD

**Issue by nickhhg** [stephenjude/filament-blog#39]
### Filament Blog Version

2

### Filament Version

3

### Laravel Version

10

### PHP Version

8.2

### Problem description

due to the foreign key constraints in the blog_posts table, you cannot reset/rollback the blog tables migration. under mysql8 you will receive the following error
```
Cannot drop table 'blog_categories' referenced by a foreign key constraint 'blog_posts_blog_category_id_foreign' on table 'blog_posts'
``` 

This can be resolved by dropping or disabling the foreign keys within the migration during rollback
https://laravel.com/docs/9.x/migrations#dropping-foreign-keys

### Expected behavior

rollbacking of blog tables migration actually works without error and blog tables are dropped

### Steps to reproduce

```
composer require stephenjude/filament-blog
php artisan filament-blog:install
php artisan migrate
php artisan migrate:rollback
``` 

### Reproduction repository

https://github.com

**Comment by stephenjude** [comment]
@nickhhg do you mind making a PR for this?

**Comment by stephenjude** [comment]
This issue has been fixed #40
